pmm-app/src/onboard/components/FeedbackTooltip/Step1.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC } from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { Button, useStyles2 } from '@grafana/ui';
+
+interface Step1Props {
+  onSubmit: () => void;
+}
+
+export const Step1: FC<Step1Props> = ({ onSubmit }) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <>
+      <div className={styles.contentTitle}>
+        How&apos;s your experience with PMM so far?
+      </div>
+      <div className={styles.rating}>
+        <Button variant="secondary" icon="sad" onClick={onSubmit}>Bad</Button>
+        <Button variant="secondary" icon="meh" onClick={onSubmit}>Fair</Button>
+        <Button variant="secondary" icon="smile" onClick={onSubmit}>Good</Button>
+      </div>
+      <div className={styles.ratingDestription}>
+        With only one click you&apos;ll help us improve PMM.
+      </div>
+    </>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  modalHeaderClose: css`
+      display: flex;
+      align-items: center;
+      justify-content: flex-end;
+  `,
+  feedbackContentForm: css`
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      padding: 0px 24px 24px 24px;
+      gap: 12px;
+
+      flex: none;
+      order: 0;
+      flex-grow: 0;
+  `,
+  contentTitle: css`
+      font-family: 'Roboto';
```

Comment:
Do we need to specify font family here? I guess it's default font for Grafana, no?